// @sfpy/node-sdk/dist/utils/constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Channel = exports.Environment = exports.CARDS_ROUTE = exports.ACCOUNTS_ROUTE = exports.DEVELOPMENT_BASE = exports.SANDBOX_BASE = exports.PROD_BASE = exports.CHECKOUT_DEVELOPMENT = exports.CHECKOUT_SANDBOX = exports.CHECKOUT_PRODUCTION = exports.URL_SANDBOX = exports.URL_PRODUCTION = exports.API_URL_DEVELOPMENT = exports.API_URL_SANDBOX = exports.API_URL_PRODUCTION = void 0;
exports.API_URL_PRODUCTION = 'https://api.getsafepay.com';
exports.API_URL_SANDBOX = 'https://sandbox.api.getsafepay.com';
exports.API_URL_DEVELOPMENT = 'https://dev.api.getsafepay.com';
exports.URL_PRODUCTION = 'https://getsafepay.com/components';
exports.URL_SANDBOX = 'https://sandbox.api.getsafepay.com/components';
exports.CHECKOUT_PRODUCTION = 'https://getsafepay.com/checkout/pay';
exports.CHECKOUT_SANDBOX = 'https://sandbox.api.getsafepay.com/checkout/pay';
exports.CHECKOUT_DEVELOPMENT = 'https://dev.api.getsafepay.com/checkout/pay';
exports.PROD_BASE = 'https://getsafepay.com';
exports.SANDBOX_BASE = 'https://sandbox.api.getsafepay.com';
exports.DEVELOPMENT_BASE = 'https://dev.api.getsafepay.com';
exports.ACCOUNTS_ROUTE = '/components';
exports.CARDS_ROUTE = '/checkout/pay';
var Environment;
(function (Environment) {
    Environment["Production"] = "production";
    Environment["Sandbox"] = "sandbox";
    Environment["Development"] = "development";
})(Environment = exports.Environment || (exports.Environment = {}));
var Channel;
(function (Channel) {
    Channel["Cards"] = "cards";
    Channel["Accounts"] = "accounts";
})(Channel = exports.Channel || (exports.Channel = {}));
