// @sfpy/node-sdk/dist/resources/checkout.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Checkout = void 0;
var builder_1 = require("../utils/builder");
var Checkout = /** @class */ (function () {
    function Checkout(config) {
        this.config = config;
    }
    Checkout.prototype.create = function (_a) {
        var cancelUrl = _a.cancelUrl, orderId = _a.orderId, redirectUrl = _a.redirectUrl, _b = _a.source, source = _b === void 0 ? 'custom' : _b, token = _a.token, _c = _a.webhooks, webhooks = _c === void 0 ? false : _c;
        var url = (0, builder_1.buildCheckoutUrl)(this.config.channel, this.config.environment);
        var params = new URLSearchParams({
            beacon: token,
            cancel_url: cancelUrl,
            env: this.config.environment,
            order_id: orderId,
            redirect_url: redirectUrl,
            source: source,
            webhooks: String(webhooks)
        }).toString();
        return "".concat(url, "?").concat(params);
    };
    return Checkout;
}());
exports.Checkout = Checkout;
